src/components/Form/Form.js

```diff
@@ -1,12 +1,26 @@
-const Form = () => (
-  <form>
-    <h3>
-      Add New Book
-    </h3>
-    <input type="text" placeholder="Book title..." />
-    <input type="text" placeholder="Author..." />
-    <button type="submit">Add Book</button>
-  </form>
-);
+import { useDispatch } from 'react-redux/es/exports';
+import { v4 as uuidv4 } from 'uuid';
+import { addBook } from '../../redux/books/books';
+
+const Form = () => {
+  const dispatch = useDispatch();
+
+  const handleSubmit = (e) => {
+    e.preventDefault();
+    dispatch(addBook(e.target.bTitle.value, e.target.bAuthor.value, uuidv4()));
+    e.target.reset();
+  };
+
+  return (
+    <form onSubmit={handleSubmit}>
+      <h3>
+        Add New Book
+      </h3>
+      <input type="text" name="bTitle" placeholder="Book title..." />
+      <input type="text" name="bAuthor" placeholder="Author..." />
```

Comment:
- [ ] The local state with `title`, `author` should have their inputs that updates the local state every time they update. Kindly add this ✍🏻 , You should have an `onChange` even, It might look like this:

```
  <input type="text" placeholder="Book Title" value={title} name="title" onChange={handleChange}  />
  <input type="text" placeholder="Author" value={author}  name="author" onChange={handleChange}  />
```

Reply:
Thanks for the feedback, I missed that requirement 😅